src/modules/edc-demo/edc-demo.module.ts

```diff
@@ -131,6 +134,9 @@ import {SafePipe} from './pipes/safe.pipe';
     SafePipe,
     TransferHistoryViewerComponent,
     TransportModeSelectComponent,
+    EmptyStateComponent,
+    ErrorStateComponent,
+    LoadingStateComponent,
```

Comment:
the above is alphabetically ordered, so we should leave it alphabetically ordered.

You can select the entire block, press CTRL+SHIFT+A and search for the "Sort Lines" action